README.md

```diff
@@ -16,4 +16,4 @@ You can reach the maintainers of this project at:
 Participation in the Kubernetes community is governed by the [Kubernetes Code of Conduct](code-of-conduct.md).
 
 [owners]: https://git.k8s.io/community/contributors/guide/owners.md
-[Creative Commons 4.0]: https://git.k8s.io/website/LICENSE
+[creative commons 4.0]: https://git.k8s.io/website/LICENSE
```

Comment:
`Creative Commons` is the right way to write the name. You'll notice https://git.k8s.io/website/LICENSE the `C` are in uppercase.

Reply:
This is a result of my linter. Reverted in e0679f7